src/testsuite/arith128_print.c

```diff
@@ -1347,6 +1364,9 @@ check_int64 (char *prefix, long val64, long shouldbe)
 
   return (rc);
 }
+
+#ifndef __clang__
+// Clang does not support _Decimal* types
```

Comment:
nit, but a blank line after this would better separate the new statement from the existing one. As is, it sounds like the TODO is to make the new comment hidden. :smile:

Reply:
Ok

#ifndef __clang__ // Clang does not support _Decimal* types

// TODO make this hidden so we can use TOC relative